R/html_document_base.R

```diff
@@ -106,7 +106,8 @@ html_document_base <- function(theme = NULL,
           options = sass::sass_options(output_style = "compressed")
         )
       }
-      f <- normalized_relative_to(output_dir, f)
+      # do not normalize web path
+      if (!xfun::is_web_path(f)) f <- normalized_relative_to(output_dir, f)
```

Comment:
I'm just wonder if it would make more sense to test the existence of the file, but I guess it would be a more risky fix (i.e., there might be cases in which the file doesn't exist given the current working directory---in particular, I'm not sure about Shiny documents):

```suggestion
      if (file.exists(f)) f <- normalized_relative_to(output_dir, f)
```

Reply:
I don't know about this either. This would require some testing to be sure. 
This transformation happens in Pandoc pre processor so right before Pandoc is ran so it should exists. 
Unless there is an edge case we missed (like order of pre processor ?) :think

Reply:
Okay. Let's just be conservative here and merge the PR. Thanks!